### ISSUE TITLE
How to with softmax loss function to train yolov3

### ISSUE THREAD

**Issue by WEITINGLIN32** [AlexeyAB/darknet#2681]
@AlexeyAB 
I face the problems that now I want to use the softmax loss to yolov3 and how do I change the yolov3.cfg to make it works? Thanks!
